prep-javascript-variables/index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>JavaScript Variables</title>
+</head>
+<body>
+  <script>
+    var fullname = "Thanh Le";
+    console.log("Value of Fullname",fullname);
+    var numberOfPets = "1";
+    console.log("value of numberOfPets",1);
+    var likesVegetables = "true";
+    console.log("value of likesVegetables",true);
+    var nothing ="null";
```

Comment:
For your number `1`, boolean `true`, and `null`, be sure to leave out the quotation marks so that JavaScript doesn't think you mean string values.